updates instructions based on RSLondon website and newer git version

alessandrofelder opened the pull request:
Closes #1

alessandrofelder committed:
updates instructions based on RSLondon website and newer git version

ageorgou commented on _pages/setup/instructions_win.md:
```diff
@@ -10,19 +10,24 @@ instructions below:
 
 1. Download the [Git for Windows installer](https://git-for-windows.github.io/).
 1. Run the installer and follow the steps below:
+Ensure that "Git Credential Manager Core" is selected and click on "Next".
+Ensure that "Enable file system caching" is selected and click on "Next".
+Click on "Install".
+Click on "Finish" or "Next".
```
These are present later on?

alessandrofelder replied:
Woops 🤦

ageorgou commented on _pages/setup/instructions_win.md:
```diff
@@ -10,19 +10,24 @@ instructions below:
    - Ensure that "Enable file system caching" is selected and click on "Next". 
-   - Experimental features: Keep the default (i.e., nothing selected) - click "Next"
    - Click on "Install".
    - Unselect "View release notes" (unless you are interested), and click on "Next". 
```
Actually, should this be `click on "Finish"` (or `"Finish or Next"`)?

alessandrofelder replied:
I've tested, and it should be "Finish" (at least according to what happens locally on native Win for me).

alessandrofelder committed:
remove silly lines

alessandrofelder committed:
make core bold, add extra "next", and "Finish"

Testing reveals there an extra step where next needs to be clicked before reaching "install".
Also, it's Finish not Next at the end.

because there is the exact same option
without "Core" at the end available to the installing person.

alessandrofelder committed:
be more explicit about experimental features

alessandrofelder commented:
Thanks for catching my silly mistakes @ageorgou ! I've manually tested the instructions locally and fixed now.
